src/conf_mode/snmp.py

```diff
@@ -383,6 +383,10 @@ def get_config():
                     oid_cfg = {
                         'oid': oid
                     }
+                    if conf.exists('v3 view {0} oid {1} exclude'.format(view, oid)):
+                        oid_cfg['exclude'] = []
+                        for exclude in conf.return_values('v3 view {0} oid {1} exclude'.format(view, oid)):
```

Comment:
Any reason to iterate of return_values and then appending them to a list?

If I‘m not misstaken, return_values() returns a list already

Reply:
Yes, it returns a list (as opposed to `return_value()`, but we may want to address this in current first and then backport the refactorings. Since it's not a logic mistake and will not have a big performance impact, I'm not really against merging it as is.

Reply:
Updated based on your comment.